Auth0/MobileWebAuth.swift

```diff
@@ -13,13 +13,13 @@ extension UIApplication {
 extension ASUserAgent: ASWebAuthenticationPresentationContextProviding {
 
 #if os(iOS)
-    func presentationAnchor(for session: ASWebAuthenticationSession) -> ASPresentationAnchor {
+   func presentationAnchor(for session: ASWebAuthenticationSession) -> ASPresentationAnchor {
         return UIApplication.shared()?.windows.last(where: \.isKeyWindow) ?? ASPresentationAnchor()
     }
 #endif
 
 #if os(visionOS)
-    func presentationAnchor(for session: ASWebAuthenticationSession) -> ASPresentationAnchor {
+    nonisolated func presentationAnchor(for session: ASWebAuthenticationSession) -> ASPresentationAnchor {
```

Comment:
Should this be `nonisolated` if this is UI-related code that needs to run on the main thread?